scripts/test.js

```diff
@@ -7,12 +7,12 @@ const testFormat = require('./helpers/test-format')
 
 const start = async () => {
   try {
+    await testFormat()
```

Comment:
I moved this to the top as it sucks to wait minutes for the other tests to pass only to find you have a formatting error